Resources/Prototypes/_Sunrise/Entities/Objects/Specific/Robotics/borg_modules.yml

```
@@ -53,7 +53,9 @@
   - type: ItemBorgModule
     items:
     - DL6902Borg
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 5

(indentation)